src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_MegaIndustrialApiary.java

```diff
@@ -517,9 +518,10 @@ public boolean checkRecipe(ItemStack aStack) {
                     for (int i = 0, mStorageSize = Math.min(mStorage.size(), mMaxSlots); i < mStorageSize; i++) {
                         BeeSimulator beeSimulator = mStorage.get(i);
                         stacks.addAll(beeSimulator.getDrops(64_00d * boosted));
+                        workingBees++;
                     }
 
-                    this.lEUt = -(int) ((double) GT_Values.V[6] * (double) mMaxSlots * 0.99d);
+                    this.lEUt = -GT_Values.VP[6] * workingBees;
```

Comment:
No need for additional variable.
```suggestion
                    this.lEUt = -GT_Values.VP[6] * mStorage.size();
```